android/build.gradle

```
@@ -29,15 +29,11 @@ android {
         namespace 'com.w3conext.jailbreak_root_detection'
     }
 
-    compileSdkVersion 33
+    compileSdkVersion 36
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
-    }
-
-    kotlinOptions {
-        jvmTarget = '11'
+        sourceCompatibility JavaVersion.VERSION_21
+        targetCompatibility JavaVersion.VERSION_21
```

```suggestion
        sourceCompatibility JavaVersion.VERSION_17
        targetCompatibility JavaVersion.VERSION_17
```

Was it necessary to upgrade to java 21? If not please consider dropping to 17 to be more inclusive